test/e2e/dsa/dsa.go

```diff
@@ -96,4 +98,26 @@ func describe() {
 
 		framework.Logf("log output: %s", log)
 	})
+
+	ginkgo.It("deploys DSA plugin with operator", func() {
```

Comment:
I wonder where's the code path the undeploys the other daemonset before running this (or vice versa)?

Reply:
@mythi 
k8s e2e deletes a namespace after the test

Reply:
Yes, but it happens *after* the test, right? As far as I understood, @mythi asked about un-deploying whatever is deployed previously in this test.